.vib/wavefront-hpa-adapter/vib-publish.json

```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/wavefront-hpa-adapter"
         },
-        "runtime_parameters": "InNlcnZpY2UiOgogICJwb3J0cyI6CiAgICAiaHR0cHMiOiA0NDMKICAidHlwZSI6ICJMb2FkQmFsYW5jZXIiCiJ3YXZlZnJvbnQiOgogICJ0b2tlbiI6ICJmYWtlIgogICJ1cmwiOiAiaHR0cHM6Ly9mYWtlLndhdmVmcm9udC5jb20i",
+        "runtime_parameters": "Y29udGFpbmVyUG9ydHM6CiAgaHR0cHM6IDY0NDQKd2F2ZWZyb250OgogIHVybDogaHR0cHM6Ly9mYWtlLndhdmVmcm9udC5jb20KICB0b2tlbjogRkFLRV9BUElfVE9LRU4Kc2VydmljZToKICB0eXBlOiBMb2FkQmFsYW5jZXIKICBwb3J0czoKICAgIGh0dHBzOiA0NDMKcmJhYzoKICBjcmVhdGU6IHRydWUKYXBpU2VydmljZToKICBjcmVhdGU6IHRydQpleHRyYURlcGxveToKLSBhcGlWZXJzaW9uOiBhdXRvc2NhbGluZy92MmJldGExCiAga2luZDogSG9yaXpvbnRhbFBvZEF1dG9zY2FsZXIKICBtZXRhZGF0YToKICAgIG5hbWU6IHZpYi1ocGEtY3VzdG9tLW1ldHJpY3MKICBzcGVjOgogICAgbWluUmVwbGljYXM6IDEKICAgIG1heFJlcGxpY2FzOiA1CiAgICBtZXRyaWNzOgogICAgLSB0eXBlOiBFeHRlcm5hbAogICAgICBleHRlcm5hbDoKICAgICAgICBtZXRyaWNOYW1lOiB2aWIudm0uY3B1CiAgICAgICAgdGFyZ2V0QXZlcmFnZVZhbHVlOiAxODAwbQogICAgc2NhbGVUYXJnZXRSZWY6CiAgICAgIGFwaVZlcnNpb246IGFwcHMvdjEKICAgICAga2luZDogRGVwbG95bWVudAogICAgICBuYW1lOiB3YXZlZnJvbnQtaHBhLWFkYXB0ZXI=",
```

Comment:
You have a typo on the params:
```
apiService:
  create: tru
```

Reply:
Oh, it is "tru" I have a typo 😆 Fixed it, thanks!